packages/ibm-products/src/components/OptionsTile/OptionsTile.stories.js

```diff
@@ -129,7 +129,7 @@ const TemplateStatic = ({ enabled, ...rest }) => {
   const [liveEnabled, setLiveEnabled] = useState(enabled);
 
   function onToggle(e) {
-    setLiveEnabled(e);
+    setLiveEnabled(!liveEnabled);
```

Comment:
`onToggle` is called with either `true` or `false` depending on what the next state should be, therefore passing it directly into the `setLiveEnabled` function was appropriate.

```suggestion
    setLiveEnabled(e);
```